ran-core/src/test/java/io/ran/IsItsOwnKeyTest.java

```diff
@@ -22,8 +22,8 @@ public class IsItsOwnKeyTest {
 
 	@Before
 	public void setup() {
-		store = new TestDoubleDb();
 		helper = new GuiceHelper();
+		store = new TestDoubleDb(new MappingHelper(helper.factory));
```

Comment:
Why not inject it now that its bound in GuideModule anyway (GuiceHelper can instatiate)